x/evm/abci.go

```diff
@@ -383,11 +417,11 @@ func validateEvent(ctx sdk.Context, event types.Event, bk types.BaseKeeper, n ty
 		return fmt.Errorf("destination chain de-activated")
 	}
 
-	if event.GetTokenSent() != nil && !types.IsEVMChain(destinationChain) {
```

Comment:
why is this check no longer necessary?

Reply:
destinationChain chain keeper validation get skipped if the dest chain is not an evm chain. 
Before we skip `tokenSent` explicitly, now it also applies to `contractCall` and `contractCallWithToken`